### ISSUE TITLE
Add -t flag to clusterd

### ISSUE THREAD

**Issue by crd1985** [wazuh/wazuh#2687]
All Wazuh daemons have the -t option to perform a configuration test, so as the wazuh service fails showing the cause. However, clusterd does not implement this option, leading to some exceptions in code like the following one:
https://github.com/wazuh/wazuh/blob/master/src/init/ossec-server.sh#L252-L254
Another great advantage, apart from code improvement and overall coherence is to avoid that the service become inaccessible after a failed restart and a bad configuration validation.

To sum up, we need the -t option available in clusterd too.
